docs/API.md

```diff
@@ -1462,86 +1579,119 @@ If the `batch_id` is undefined the following error is returned:
 }
 ```
 
-## API method: `get_test_params`
 
-Return a normalized *params* objects of a *test*.
+## Deprecated
 
-Example request:
+### API method: `version_info`
 
-*Valid syntax:*
-```json
-{
-    "jsonrpc": "2.0",
-    "id": 143014426992009,
-    "method": "get_test_params",
-    "params": {"test_id": "6814584dc820354a"}
-}
-```
+**Deprecated**. Same as [`system_versions`][system_versions].
 
-Example response:
-```json
-{
-    "jsonrpc": "2.0",
-    "id": 143014426992009,
-    "result": {
-         "domain": "zonemaster.net",
-         "profile": "default",
-         "client_id": "Zonemaster Dancer Frontend",
-         "nameservers": [
-            {
-                "ns": "ns3.nic.se",
-                "ip": "2001:67c:124c:2007::45"
-            },
-            {
-                "ip": "192.93.0.4",
-                "ns": "ns2.nic.fr"
-            }
-         ],
-         "ipv4": true,
-         "ipv6": true,
-         "client_version": "1.0.1",
-         "ds_info": []
-    }
-}
-```
 
+### API method: `profile_names`
 
-#### `"params"`
+**Deprecated**.
+Same as [`conf_profiles`][conf_profiles], but returns only the array of
+[*Profile names*][Profile name].
```

Comment:
Unclear, what does "but returns only the array of Profile names" mean?

Reply:
Note added.